ci/test/00_setup_env_native_nowallet.sh

```diff
@@ -7,8 +7,8 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_native_nowallet
-export DOCKER_NAME_TAG=ubuntu:18.04  # Use bionic to have one config run the tests in python3.6, see doc/dependencies.md
-export PACKAGES="python3-zmq clang-5.0 llvm-5.0"  # Use clang-5 to test C++17 compatibility, see doc/dependencies.md
+export DOCKER_NAME_TAG=ubuntu:20.04
```

Comment:
Would be nice to not change this. clang-7 is also in bionic: 

* https://packages.ubuntu.com/bionic-updates/clang-7

Reply:
Still not addressed?

Reply:
That's why it hasn't been marked as resolved.

Reply:
This has now been dropped.